pkg/verifier/aws/aws_verifier.go

```diff
@@ -179,6 +179,12 @@ func (a *AwsVerifier) createEC2Instance(input createEC2InstanceInput) (string, e
 		return "", handledErrors.NewGenericError(err)
 	}
 
+	// Wait up to 5 minutes for the instance to be running
+	waiter := ec2.NewInstanceRunningWaiter(a.AwsClient)
```

Comment:
What led to this being added?

Reply:
Ah it's a native `aws-sdk-go-v2` way to "wait" for things to happen https://aws.github.io/aws-sdk-go-v2/docs/making-requests/#using-waiters which let us deprecate the custom logic we had previously in `WaitForEC2InstanceCompletion` and `describeEC2Instances`

Reply:
The cost being a permissions switch from `ec2:DescribeInstanceStatus` to `ec2:DescribeInstances`

Reply:
@mjlshen Makes sense. However, I think this should be done in a separate PR.

Reply:
Done in #149